docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.module

```diff
@@ -700,7 +700,12 @@ function cgov_core_entity_build_defaults_alter(array &$build, EntityInterface $e
  * Implements field_widget_single_element_WIDGET_TYPE_form_alter().
  */
 function cgov_core_field_widget_single_element_link_default_form_alter(&$element, FormStateInterface $form_state, $context) {
-  $element['uri']['#description'] = t('Enter the full URL starting with https://');
+  if ($element['#title'] == "Audio-Described Video Link") {
```

Comment:
As discussed at standup, this needs to be moved to `cgov_video`, as this field does not exist without that. Also you could probably target the field name instead of the label? You should not have 20 different audio described fields to make this be generic. The current logic is because **ANY** external field, and you know we have a lot, removes things like autocomplete, and gets this overridden text.

Finally, order will matter. The code in the cgov_video _alter should fire after this. But just know that each hook is fired off one after the other based on the modules weight.